### DISCUSSION TITLE
Build Failed Command "yarn run build" exited with 1

### DISCUSSION THREAD

**Discussion by karengiovanna** [vercel/next.js#27089] [Help]
14:56:08.365  	Cloning github.com/karengiovanna/Podcaster (Branch: main, Commit: 422021c)
14:56:08.915  	Cloning completed: 549.928ms
14:56:08.929  	Analyzing source code...
14:56:09.907  	Installing build runtime...
14:56:17.351  	Build runtime installed: 7.444s
14:56:20.429  	No Build Cache available
14:56:21.521  	Installing dependencies...
14:56:21.877  	yarn install v1.22.10
14:56:21.933  	warning package-lock.json found. Your project contains lock files generated by tools other than Yarn. It is advised not to mix package managers in order to avoid resolution inconsistencies caused by unsynchronized lock files. To clear this warning, remove package-lock.json.
14:56:21.936  	[1/4] Resolving packages...
14:56:22.113  	[2/4] Fetching packages...
14:56:27.720  	info fsevents@2.3.2: The platform "linux" is incompatible with this module.
14:56:27.720  	info "fsevents@2.3.2" is an optional dependency and failed compatibility check. Excluding it from installation.
14:56:27.725  	[3/4] Linking dependencies...
14:56:30.440  	[4/4] Building fresh packages...
14:56:30.456  	Done in 8.58s.
14:56:30.475  	Detected Next.js version: 10.2.3
14:56:30.477  	Running "yarn run build"
14:56:30.746  	yarn run v1.22.10
14:56:30.773  	$ next build
14:56:31.463  	info  - Using webpack 5. Reason: no custom webpack configuration in next.config.js https://nextjs.org/docs/messages/webpack5
14:56:31.593  	Attention: Next.js now collects completely anonymous telemetry regarding usage.
14:56:31.594  	This information is used to shape Next.js' roadmap and prioritize features.
14:56:31.594  	You can learn more, including how to opt-out if you'd not like to participate in this anonymous program, by visiting the following URL:
14:56:31.594  	https://nextjs.org/telemetry
14:56:31.631  	info  - Checking validity of types...
14:56:34.555  	Failed to compile.
14:56:34.556  	./src/components/Header/index.tsx:1:20
14:56:34.556  	Type error: Cannot find module 'date-fns/format' or its corresponding type declarations.
14:56:34.556  	> 1 | import format from 'date-fns/format';
14:56:34.556  	    |                    ^
14:56:34.556  	  2 | import ptBR from 'date-fns/locale/pt-BR';
14:56:34.556  	  3 | import styles from './styles.module.scss';
14:56:34.556  	  4 | 
14:56:34.578  	error Command failed with exit code 1.
14:56:34.578  	info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
14:56:34.595  	Error: Command "yarn run build" exited with 1

[Solve this Issue](https://github.com/karengiovanna/Podcaster/issues/2)

**Comment by guitheengineer** [comment]
Maybe is just lacking the date-fns package? 
`yarn add date-fns` should solve it.

If the problem persists, you can also try a named import
`import { format } 'date-fns'`

**Reply by guitheengineer** [reply]
Also, don't install @types/date-fns, date-fns already has built-in types, without the need for this package (which also has been deprecated https://www.npmjs.com/package/@types/date-fns)

**Reply by guitheengineer** [reply]
Update: tried `yarn build` on your repo and it seems like this problem is solved. Note that this isn't related to nextjs though.
